internal/cocoasdk/nsinvocation_darwin.go

```diff
@@ -0,0 +1,46 @@
+package cocoasdk
+
+import (
+	"github.com/ebitengine/purego/objc"
+	"unsafe"
+)
+
+var class_NSInvocation = objc.GetClass("NSInvocation\x00")
```

Comment:
Actually we don't have to use `\x00` but we use `\x00` for performance, right?

If the performance really matters, it's OK, but I feel like this degrates the readability. I prefer not having `\x00`. What do you think?

Reply:
I didn't notice any major differences. Ill remove them